pkg/trivy/wrapper.go

```diff
@@ -54,14 +53,19 @@ func (w *wrapper) Run(imageRef string, auth RegistryAuth) (report trivy.ScanRepo
 		}
 	}()
 
-	cmd := exec.Command(executable,
-		"--quiet",
+	args := []string{
+		"--no-progress",
+		"--debug",
```

Comment:
nit: `--debug` is always added? We can add it only when the environment variable is set.

Reply:
Good point. Thanks. I've added env to conditionally add the `--debug` flag.